src/pages/normal/normal.ts

```diff
@@ -12,60 +12,59 @@ export class NormalPage {
   constructor(public navCtrl: NavController, public navParams: NavParams, private shareService: ShareService, private storage: Storage) {
   }
 
-  possible: Array<any> = [
+  readonly possible: Array<string> = [
   "AliceBlue","Aqua","Aquamarine","Azure","Beige","Black","Blue","BlueViolet","Brown","BurlyWood","CadetBlue","Chartreuse","Chocolate","Coral","CornflowerBlue","Crimson","Cyan","DarkBlue","DarkCyan","DarkGoldenRod","DarkGrey","DarkGreen","DarkKhaki","DarkMagenta","DarkOliveGreen","DarkOrange","DarkOrchid","DarkRed","DarkSalmon","DarkSeaGreen","DarkSlateBlue","DarkSlateGrey","DarkTurquoise","DarkViolet","DeepPink","DeepSkyBlue","DimGrey","DodgerBlue","FireBrick","FloralWhite","ForestGreen","Fuchsia","Gainsboro","GhostWhite","Gold","GoldenRod","Grey","Green","GreenYellow","HotPink","IndianRed","Indigo","Ivory","Khaki","Lavender","LavenderBlush","LawnGreen","LemonChiffon","LightBlue","LightCoral","LightCyan","LightGrey","LightGreen","LightPink","LightSalmon","LightSeaGreen","LightSkyBlue","LightSlateGrey","LightSteelBlue","LightYellow","LimeGreen","Magenta","Maroon","MidnightBlue","Olive","Orange","OrangeRed","Orchid","PaleGreen","PaleTurquoise","PaleVioletRed","Pink","Purple","Red","RoyalBlue","SaddleBrown","Salmon","SeaGreen","Silver","SkyBlue","Snow","SpringGreen","SteelBlue","Teal","Tomato","Turquoise","Violet","White","WhiteSmoke","Yellow","YellowGreen"
   ];
   point: number = 0;
   bestPoint: boolean;
-  nbColors: number[] = [2, 4, 6, 8];
+  readonly nbColors: number[] = [2, 4, 6, 8];
   difficulty: number = 0;
-  hexa: string = "#";
-  goodColor: string ="";
+  goodColor: string = "";
   color: string;
-  colors: Array<any> = [];
+  colors: Array<string> = [];
   fail: number = 0;
   testcolor: boolean;
   record: number;
-  timer: any;
+  timer: number;
```

Comment:
Set timer to number send an error (any work)

Reply:
ok le truc c'est ca marche chez moi @cle61 meme probleme de ton cote ?

@Gaviil tu peux me donner stp les resultats de 

```bash
node -v
tsc -v

Reply:
pour ma part

```bash
node -v
    8.6.0

tsc -v
    2.5.2

Reply:
Bon bah je viens de faire une reinstall sur un autre PC et ca marche la aussi ;(

Reply:
```bash
node -v
    v6.11.2

tsc -v
    j'ai pas donc le soucis est surement chez moi

Reply:
et avec ./node_modules/typescript/bin/tsc ?

la commande ionic je la lance comme ca ./node_modules/ionic/bin/ionic serve -l

Reply:
pas tres actifs ce projet :(

Reply:
Pas très actif, je cherche pourquoi j'ai de la merde ...
@cle61 toi tout est ok ?

Reply:
Je testerai ce week end :)